apps/aens/src/aens_claim_tx.erl

```diff
@@ -196,7 +234,7 @@ account_pubkey(#ns_claim_tx{account_id = AccountId}) ->
 
 -spec version(tx()) -> non_neg_integer().
 version(_) ->
```

Comment:
This must be depending on  height

Reply:
Please make all tests pass and compatible with synchronizing to an existing chain.

Reply:
Done